src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ReleaseDateServiceTest.kt

```diff
@@ -352,7 +352,90 @@ class ReleaseDateServiceTest {
         conditionalReleaseDate = null,
       )
 
-      assertThat(service.isInHardStopPeriod(licence, now)).isFalse()
+      assertThat(service.isInHardStopPeriod(licence, now)).isFalse
+    }
+  }
+
+  @Nested
+  inner class `Licence is due to be released in the next two days` {
+    @Test
+    fun `licence is not due to be released in the next two days if there is no release date`() {
+      val licence = createCrdLicence().copy(
+        actualReleaseDate = null,
+        conditionalReleaseDate = null,
+      )
+
+      assertThat(service.isDueToBeReleasedInTheNextTwoWorkingDays(licence)).isFalse
+    }
+
+    @Test
+    fun `licence is not due to be released in the next two working days when CRD is yesterday`() {
+      val licence = createCrdLicence().copy(
+        actualReleaseDate = null,
+        conditionalReleaseDate = LocalDate.now(clock).minusDays(1),
+      )
+
+      assertThat(service.isDueToBeReleasedInTheNextTwoWorkingDays(licence)).isFalse
+    }
+
+    @Test
+    fun `licence is due to be released in the next two working days when CRD is today`() {
+      val licence = createCrdLicence().copy(
+        actualReleaseDate = null,
+        conditionalReleaseDate = LocalDate.now(clock),
+      )
+
+      assertThat(service.isDueToBeReleasedInTheNextTwoWorkingDays(licence)).isTrue
+    }
+
+    @Test
+    fun `licence is due to be released in the next two working days when CRD is 1 day in the future`() {
+      val licence = createCrdLicence().copy(
+        actualReleaseDate = null,
+        conditionalReleaseDate = LocalDate.now(clock).plusDays(1),
+      )
+
+      assertThat(service.isDueToBeReleasedInTheNextTwoWorkingDays(licence)).isTrue
+    }
+
+    @Test
+    fun `licence is due to be released in the next two working days when CRD is 2 working days in the future`() {
+      val licence = createCrdLicence().copy(
+        actualReleaseDate = null,
+        conditionalReleaseDate = LocalDate.now(clock).plusDays(3),
```

Comment:
Should this not be `.plusDays(2)`?

Also perhaps more importantly, should this not be failing in its current state?

Reply:
hate these tests but now == `2023-11-10T00:00:00Z` which is a Saturday 
So 3 days is 2 working days

Reply:
I've rewritten these